src/main/assets/js/admin-filtering.js

```diff
@@ -138,43 +134,39 @@ angular.module('slicebox.adminFiltering', ['ngRoute'])
             filterSpec: null
         };
 
-        $scope.attributeActions = [
+        $scope.tagPathActions = [
             {
                 name: 'Remove',
-                action: removeAttributes
+                action: removeTagPaths
             }
         ];
 
         $scope.$watch('uiState.selectedFilter', function() {
             resetState();
         });
 
-        $scope.addFilterAttributeButtonClicked = function() {
-            var newAttribute = {};
-
+        $scope.addFilterTagPathButtonClicked = function() {
             var dialogPromise = $mdDialog.show({
-                templateUrl: '/assets/partials/editFilterAttributeModalContent.html',
-                controller: 'EditSeriesTypeRuleAttributeModalCtrl',
+                templateUrl: '/assets/partials/editFilterTagPathModalContent.html',
+                controller: 'EditFilterTagPathModalCtrl',
                 locals: {
-                    attribute: newAttribute
+                    tagPath: {}
                 }
             });
 
-            dialogPromise.then(function () {
-                if ($scope.state.filterSpec.tags.map(function(t) {return t.tag;}).includes(newAttribute.tag)) //Don't include tag twice
-                    return;
-                $scope.state.filterSpec.tags.push(newAttribute);
-                $scope.state.filterSpec.tags = $scope.state.filterSpec.tags.sort(function(a, b) {return a.tag - b.tag;});
-                saveFilter($scope.state.filterSpec).then(function() {
-                    $scope.callbacks.filterAttributesTables.reloadPage();
```

Comment:
Had to move checks for uniqueness and sorting to the backend since this is more complicated for tag paths